app/models/salary_graph.rb

```diff
@@ -8,30 +8,17 @@ def initialize(employees, dates)
 
   def to_table
     dates.map do |date|
-      # Multiply by 1000 to convert from seconds since the epoch to
-      # milliseconds since the epoch
-      format_date(date) + employees_row_for(date)
+      employees_row_for({ date: date })
```

Comment:
We could simplify this by just passing the date and making the hash in employees_row_for instead of here.